src/pages/AddLiquidity/index.tsx

```diff
@@ -304,6 +304,14 @@ export default function AddLiquidity() {
 
   const isCreate = location.pathname.includes('/create')
 
+  useEffect(() => {
+    //input cleanup on component unmount
```

Comment:
Do we need the comment?

I would remove it. 
If we want to to be descriptive I would suggest make it a function and give it a descriptive name eg. 
`cleanUpInput`

Reply:
Updated to a named function